app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccinated = True
+    mask_not_wear_count = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            vaccinated = False
+        except NotWearingMaskError:
+            mask_not_wear_count += 1
+
+    if not vaccinated:
+        return "All friends should be vaccinated"
+    elif mask_not_wear_count > 0:
+        return f"Friends should buy {mask_not_wear_count} masks"
+    else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`